paddle/go/pserver/service.go

```diff
@@ -0,0 +1,179 @@
+package pserver
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+)
+
+// ElementType is the type of elements of a Parameter.
+type ElementType int
+
+var ErrUnintialized = errors.New("pserver not initialized")
```

Comment:
I think errors can not be used as constants: https://golang.org/src/io/io.go `var EOF = errors.New("EOF")`